### ISSUE TITLE
Error when selecting Angel's Infinite Ores

### ISSUE THREAD

**Issue by undarl** [masterpoi/MinerPlanner#1]
Heya.

This mod is a thing of beauty. Congrats!

It currently errors out when selecting an ore patch that includes infinite ore resources from Angel's Infinite Ores.

```
Error while running event
on_player_selected_area(ID 49)
Unknown item "infinite-angels-ore4".
stack traceback:
__Miner_Planner__/control.lua:359:in function 'remote_show_gui' 
__Miner_Planner__/control.lua:67: in function 'remote_on_player_selected_area'
__Miner_Planner__/control.lua:591: in function <__Miner_Planner__/control.lua:590>
```

**Comment by masterpoi** [comment]
Thanks! It appears that that ore doesn't have an icon. I guess I will need to find the matching non infinite ore and use that one. Also 
need to check if the mod works with mixed ore patches

**Comment by Liquid5n0w** [comment]
This fix would help us a lot in our game to build on our patches!

**Comment by masterpoi** [comment]
Fixed  in 0.9.7
